pkg/app/api/service/pipedservice/service.proto

```diff
@@ -149,10 +156,18 @@ enum ListOrder {
 }
 
 message PingRequest {
-    pipe.model.PipedStats piped_stats = 1 [(validate.rules).message.required = true];
+    pipe.model.PipedStats piped_stats = 1 [(validate.rules).message.required = true, deprecated = true];
 }
 
 message PingResponse {
+    int64 ping_interval = 1 [deprecated = true];
+}
+
+message ReportStatRequest {
+    bytes piped_stats = 1 [(validate.rules).bytes.min_len = 1];
```

Comment:
Nice. For those who look at this first will be more likely to wonder why it's represented as a byte sequence. I feel like it should be commented like: "Metrics byte sequence in OpenMetrics format"

Reply:
👍 Lets me adopt your idea, thx 🙆‍♂️

Reply:
addressed by https://github.com/pipe-cd/pipe/pull/2181/commits/2bb3bcb399c6d2f09f3bc0b8417038270b8829b3 🙏